src/reconstruction/modelGeneration/massBalance/checkMassChargeBalance.m

```diff
@@ -240,6 +240,9 @@
     imBalancedRxnBool = imBalancedRxnBool | imBalancedCharge ~= 0;
 end
 
+if isfield(model,'SIntRxnBool')
+    imBalancedRxnBool = imBalancedRxnBool | ~model.SIntRxnBool;
```

Comment:
in line 87, model.SIntRxnBool is already accessed, so if its not a field there, there will be an error. Also in line 55, its actually initialized if it is not present.